components/raftstore/src/store/worker/region.rs

```diff
@@ -602,6 +605,14 @@ where
                     DeleteStrategy::DeleteByRange,
                     &CLEAR_OVERLAP_REGION_DURATION.by_range,
                 )
+            // Meanwhile, if `forcely_by_key` is specified and
+            // `cfg.use_delete_range` is not enabled, it should remove the
+            // range by delete keys rather than ingestion.
+            } else if forcely_by_key {
+                (
+                    DeleteStrategy::DeleteByKey,
+                    &CLEAR_OVERLAP_REGION_DURATION.by_key,
+                )
             } else {
                 (
                     DeleteStrategy::DeleteByWriter {
```

Comment:
Looks like it becomes dead code after this PR, could you remove it?

Reply:
Maybe we can add additional annotations here for reviewers and developers, as this approach could be applied to other delete operations if necessary.